integrations/anthropic/src/haystack_integrations/components/generators/anthropic/chat/chat_generator.py

```diff
@@ -414,7 +414,29 @@ def _convert_streaming_chunks_to_chat_message(
 
         # Update meta information
         last_chunk_meta = chunks[-1].meta
-        usage = self._get_openai_compatible_usage(last_chunk_meta)
+
```

Comment:
Looks good but let's just create first_chunk_meta = chunks[0].meta so we can use it multiple times below. While we guarantee that chunks is not None perhaps we should add a check if chunks conditional expression here.